pkg/util/version/version_test.go

```diff
@@ -79,3 +79,9 @@ func TestVersionMatchesModifier(t *testing.T) {
 	matchVersion := "2.3.2+cu118"
 	require.True(t, Matches(version, matchVersion))
 }
+
+func TestGreaterThanOrEqualToWithInvalidPatch(t *testing.T) {
+	leftVersion := "1.1.0b2"
+	rightVersion := "1.1.0b2"
+	require.True(t, GreaterOrEqual(leftVersion, rightVersion))
+}
```

Comment:
These aren't invalid patches... they need to be handled... We use [go-pep440-version](https://github.com/aquasecurity/go-pep440-version) elsewhere to do this. How about we use this here?

Reply:
Resolved in [422a29d](https://github.com/replicate/cog/pull/2415/commits/422a29dc48b2edee5569d3408edb3ee8519d4da2)